### PR TITLE
fixed peer test bugs

### PR DESCRIPTION
**Pull request by omenUchiha**

### CODE REVIEW

File: src/modules/components/message-boxes-components/MessageBox.tsx [hunk 1]
```diff
@@ -211,15 +214,15 @@ const StringBox = ({
...
-            state={{
-              communityId: userContext.community.id,
-              memberId: userId,
-            }}
+          <div
+          // to={directMessageInfoPath}
+          // state={{
+          //   communityId: userContext.community.id,
+          //   memberId: userId,
+          // }}
```
Comment (jaixchaudhary) [inline src/modules/components/message-boxes-components/MessageBox.tsx:222]: remove these comments

File: src/modules/components/message-boxes-components/MessageBox.tsx [hunk 2]
```diff
@@ -89,7 +89,10 @@ const MessageBoxDM = ({
           <>
             {parse(linkConverter(tagExtracter(messageString, userContext)))}
             {conversationObject?.state === 19 &&
-            generalContext?.currentChatroom?.chat_request_state === 2 ? (
+            generalContext?.currentChatroom?.chat_request_state === 2 &&
+            userContext.currentUser.id ===
+              generalContext.currentChatroom.chat_requested_by[0].id &&
+            index === chatroomContext.conversationList.length - 1 ? (
```
Comment (jaixchaudhary) [inline src/modules/components/message-boxes-components/MessageBox.tsx:95]: Add comments on explaining these complex logics

File: src/modules/hooks/firebase.ts [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
   const params = useParams();
   const id: any = params[routeVariable.id];
-  const mode: any = params[routeVariable.mode];
-  const operation: any = params[routeVariable.operation];
   useEffect(() => {
     const query = ref(db, `collabcards`);
 
     return onValue(query, (snapshot: any) => {
       if (snapshot.exists()) {
+        log("snap aya h ");
```
Comment (jaixchaudhary) [inline src/modules/hooks/firebase.ts:20]: why this log

File: src/modules/components/message-boxes-components/MessageBox.tsx (hunk 1 above)
Comment (omenUchiha) [reply src/modules/components/message-boxes-components/MessageBox.tsx]: removed

File: src/modules/components/message-boxes-components/MessageBox.tsx (hunk 2 above)
Comment (omenUchiha) [reply src/modules/components/message-boxes-components/MessageBox.tsx:96]: added

File: src/modules/hooks/firebase.ts (hunk 3 above)
Comment (omenUchiha) [reply src/modules/hooks/firebase.ts]: removed

**Review by jaixchaudhary — APPROVED** [review]

**Comment by omenUchiha** [comment]
https://likemindscommunity.atlassian.net/browse/LM-8992

**Pull request merged**